content/content-zh-cn.md

```diff
@@ -0,0 +1,449 @@
+    <img src="img/img-components-4-2.png" class="img-component">
+    <img src="img/img-components-4-3.png" class="img-component">
+    <img src="img/img-components-4-4.png" class="img-component">
+    <img src="img/img-components-4-5.png" class="img-component">
+    <img src="img/img-components-4-6.png" class="img-component">
+    <img src="img/img-components-4-7.png" class="img-component">
+    <img src="img/img-components-5-0.png" class="img-component">
+    <img src="img/img-components-5-1.png" class="img-component">
+    <img src="img/img-components-5-2.png" class="img-component">
+    <img src="img/img-components-5-3.png" class="img-component">
+    <img src="img/img-components-5-4.png" class="img-component">
+    <img src="img/img-components-5-5.png" class="img-component">
+    <img src="img/img-components-5-6.png" class="img-component">
+    <img src="img/img-components-5-7.png" class="img-component">
+    <img src="img/img-components-6-0.png" class="img-component">
+    <img src="img/img-components-6-1.png" class="img-component">
+    <img src="img/img-components-6-2.png" class="img-component">
+    <img src="img/img-components-6-3.png" class="img-component">
+    <img src="img/img-components-6-4.png" class="img-component">
+    <img src="img/img-components-6-5.png" class="img-component">
+    <img src="img/img-components-6-6.png" class="img-component">
+    <img src="img/img-components-6-7.png" class="img-component">
+    <img src="img/img-components-7-0.png" class="img-component">
+    <img src="img/img-components-7-1.png" class="img-component">
+    <img src="img/img-components-7-2.png" class="img-component">
+    <img src="img/img-components-7-3.png" class="img-component">
+    <img src="img/img-components-7-4.png" class="img-component">
+    <img src="img/img-components-7-5.png" class="img-component">
+    <img src="img/img-components-7-6.png" class="img-component">
+    <img src="img/img-components-7-7.png" class="img-component">
+</div>
+</div>
+
+对于实际的JPEG图像来说，这就是基本原理，剩下的只有一些额外的细节。
+
+图像被分解为8x8块，每个块分别进行分解。我们使用一组频率来确定每个像素的亮度或暗度，然后是另外两组用于颜色，一组用于红绿色，另一组用于蓝黄色。我们为每个块使用的频率个数决定了JPEG图像的品质。
+
+这是一个实际的JPEG图像，放大后我们可以看到细节。当我们改变JPEG品质水平时，可以观察出画质的区别。
+
+<div id="jpeg-example" class="sketch">
+    <img src="img/cat.png" class="sketch-child clear-pixels">
+</div>
+
+## 总结
+
+让我们回顾一下：
+
+- 傅里叶变换让我们输入一个事物，并将其分解为不同频率的成分
+- 频率告诉我们有关数据的一些基本属性
+- 并且可以通过仅存储重要的成分来压缩数据
+- 我们还可以用傅里叶变换的原理，通过一堆圆圈制作看起来很酷的动画
+
+这只是表面上的一些浅层次应用。傅里叶变换是一个非常强大的工具，因为将事物分解成不同频率是十分重要的分析方法。它们被用于许多领域，包括电路设计，移动网络信号，磁共振成像（MRI）和量子物理！
+
+## 一些问题
+
+我在这里跳过了大部分的数学原理。如果你对它的数学原理很感兴趣，可以用以下这些问题来帮助你研究：
+
+- 你如何在数学上表示傅里叶变换？
+- 连续时间傅立叶变换和离散时间傅立叶变换之间有什么区别？
+- 你如何计算傅里叶变换？
+- 你如何对整首歌曲进行傅里叶变换？ （不仅仅是单个音符）
+
+## 拓展阅读
+
+要了解更多信息，你可以看看这些非常好的资源：
+
+[An Interactive Guide To The Fourier Transform](https://betterexplained.com/articles/an-interactive-guide-to-the-fourier-transform/)  
+A great article that digs more into the mathematics of what happens.
```

Comment:
FYI, this text is still in English. The part in the [] should stay the same because that's the title, but you might want to update the description. Also possibly worth mentioning it's in English.